### ISSUE TITLE
NTR: Anti-MDA5-antibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5805]
**Preferred term label:**
Anti-MDA5-antibody positivity

**Synonyms**
Increased level of Anti-melanoma differentiation-associated proteine 5

**Definition (free text, please give PubMed ID)**
Definition:  The presence of autoantibodies in the serum that react to Anti-MDA5 (Anti-melanoma differentiation-associated proteine 5).

Comment: Anti-MDA5-antibodies are myositis‐specific autoantibodies, that are specific for clinically amyopathic dermatomyositis (CADM), an uncommon presentation of idiopathic inflammatory myositis, which is associated with rapidly progressive interstitial lung disease (RP-ILD). Most patients showed histopathological findings with diffuse alveolar damage (DAD) and poor prognosis. Furthermore it was found that heterozygote variants in IFIH1, result in gain of function of MDA5 which cause a spectrum of neuro-immunological features consistently associated with an enhanced interferon state. Type I Interferonopathy can occur by this gain of function mechanism, leading to enhanced sensitivity or consitutive activation of a nucleic acid reporter and thereby altering the threshold at which endogenous nucleic acid is sensed. 
 
PMID: 32270621 DOI:10.1111/imj.14789
doi:10.1038/ng.2933
https://doi-org.emedien.ub.uni-muenchen.de/10.1146/annurev-immunol-042718-041257
https://doi.org/10.1016/j.rmcr.2016.05.008

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Gene associated**
IFIH1

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:51   (Disorder)   Aicardi-Goutières syndrome
ORPHA:85191   (Disorder)   Singleton-Merten dysplasia
ORPHA:182095   (Group of disorders)   Interstitial lung disease

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
added as HP:0033033
